src/main/java/org/reaktivity/nukleus/oauth/internal/Capabilities.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.oauth.internal;
+
+public class Capabilities
+{
+    private static final int CHALLENGE_MASK = 0x01;
```

Comment:
Initial value should be `1 << Capability.CHALLENGE.ordinal()` instead of `0x01`.